### DISCUSSION TITLE
set temperature should explicitly say it is liquid temperature and not CPU

### DISCUSSION THREAD

**Discussion by msLinuxNinja** [liquidctl/liquidctl#555] [General]
The command `liquidctl set temperature 20 30 25 60` changes fan speed based on liquid temperature. The homepage documentation should explicitly say temperature is liquid temperature as it could be confused with CPU temperature.

**Comment by jonasmalacofilho** [comment]
I agree that in most cases it happens to be the liquid temperature.

But no, the `set <channel> speed <profile>` command doesn't necessarily set speeds based on liquid temperature. That depends on the device and, potentially (although not yet in practice), the channel in question.

And the sensor used (or used by default) is already mentioned in the device-specific documentation.

---

All that said, I'll add a note to the README mentioning that, generally, in liquid coolers that sensor tends to be the embedded liquid temperature probe. But I doubt many people will get to that part, much less actually read & remember it afterwards...

EDIT: done in 57e270168c13.

**Comment by msLinuxNinja** [comment]
> But I doubt many people will get to that part, much less actually read

True, but some weirdos like myself do read the documentation 😉  thanks for the update!
